### ISSUE TITLE
Sqlite3 dependencies

### ISSUE THREAD

**Issue by tennisparty** [Motion-Project/motionplus#20]
0. Reviewed guide and contributing documents? (Yes/No):No
1. version [x.y.z, hash, other]:[2217a13]
2. installed as a package or compiled from sources [deb, rpm, git, other]:git
3. video stream source [V4L2, network camera(http, rtsp, etc), mmal]:-
4. hardware [x86, ARM, other]:pi3A+
5. operating system [18.04, Stretch, etc, FreeBSD, other]:bullseye

I am trying to install motion plus to test an ov9281 on a pi. I have installed all dependencies listed in the INSTALL file. When I try to configure I get the error - 'Required SQLite3 packages were not found. Please check motion plus_guide.html and install necessary dependencies'

I have tried installing sqlite3 with apt install but this does not help.

Any guidance appreciated, keep up the good work,
Sam

**Comment by Mr-Dave** [comment]
Try ```sudo apt-get install libsqlite3-dev```

Note that this app has not been stable since the addition of the SQLite3 so significant revisions are going to be forthcoming to change the DBMS.   In the interim, make sure the target_dir is only local, SQLite3 isn't being used elsewhere in MotionPlus, and I'd suggest a cron job to kill MotionPlus and restart it periodically.  

(SQLite3 frequently hangs when used on mounted or network drives and then that causes a watchdog timeout which cascades to lots of other bad things)

**Comment by tennisparty** [comment]
Thanks that worked, it's all installed. I was trying to use an ov9281 sensor (as I was hoping it might be up with libcamera) but am getting 
'[NTC][ALL][00:motionplus] main: Motion camera 1 restart
[NTC][VID][01:ml01:] mlp_cam_start: Opening V4L2 device
[NTC][VID][01:ml01:] v4l2_device_open: Opening video device /dev/video0
[NTC][VID][01:ml01:] v4l2_set_input: Name = "unicam-image"- CAMERA
[NTC][VID][01:ml01:] v4l2_set_norm: Device does not support specifying PAL/NTSC norm
[NTC][VID][01:ml01:] v4l2_pixfmt_try: Unable to use palette YU12 (640x480)
[NTC][VID][01:ml01:] v4l2_palette_set: Configuration palette index 17 (YU12) for 640x480 doesn't work.'

I'll wait for the docs. Thanks for the help :)

**Comment by Mr-Dave** [comment]
You can use ```libcamerify motionplus``` or use traditional motion with ```libcamerify motion``` and it will create the v4l2 device from the camera for use in MotionPlus.

If you want to use the libcamera directly, MotionPlus does support it by specifying 
```libcam_name camera0```
in the configuration file.

**Comment by tennisparty** [comment]
Thankyou that works for me, awesome, I did have to rename the conf file as it had been created but with a different name. I'm getting quite a lot of lag using the ov9281 when viewing the camera in the web portal. For comparison I've tried the very basic picamera2 mjpeg example streaming code and and that streams with no noticeable lag to a webserver, if I can work it out I might try implementing motion on a low resolution stream with that. If there are any recommended settings to try with motion to reduce lag so let me know and I'll try them out, thanks again, awesome project
